google-cloud-clients/google-cloud-contrib/google-cloud-spanner-jdbc/src/main/java/com/google/cloud/spanner/jdbc/ReadWriteTransaction.java

```diff
@@ -0,0 +1,1283 @@
+/*
```

Comment:
This is a really big class. Is it possible to break out some of the package private classes into their own files?

Reply:
I've moved the retriable queries and updates to separate class files. That significantly reduces the size of the `ReadWriteTransaction` file.